src/numbers.js

```diff
@@ -1,45 +1,52 @@
 function add (a, b) {
-  // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
-  // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
-  // your code here
+  return a * b;
 }
 
 function divide (a, b) {
-  // your code here
+  return a / b;
 }
 
 function power (a, b) {
-  // your code here
+  return a ** b;
 }
 
 function round (a) {
-  // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
-  // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
-  // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
-  // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
-  // your code here
+  let solution;
+  if (a > 0 && b > 0) {
+    solution = Math.floor(a / b);
+  }
+  else {
+    solution = Math.ceil(a / b);
+  }
+  return solution;
```

Comment:
Theres another built in function for this: Math.trunc (see [docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc))
```
  return Math.trunc(a / b);
```